src/backend/core/migrations/0009_alter_user_language_and_sub.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.1.2 on 2024-11-08 09:11
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0008_alter_document_link_reach'),
+    ]
```

Comment:
You don't need to add a migration. This is not impacting the db schema, only validation in the Django code.